CHANGELOG.md

```diff
@@ -1,4 +1,33 @@
 ## 4.16.0 (Unreleased)
+BREAKING CHANGES:
```

Comment:
```suggestion
NOTE: We're marked a change in this release as a `BREAKING CHANGE` to indicate that the change may cause undesirable behavior for users in some circumstances. This is done to increase visibility on the change, which otherwise would have been marked under the `BUG FIXES` category, and it is not believed to be a change that breaks the backwards compatibility of the provider requiring a major version change.

BREAKING CHANGES:
```

Reply:
@megan07 @ScottSuarez: We got some feedback at one point where users were confused why we were marking changes as BCs in minors. I'm a fan of the practice- it increases visibility on risky or technically breaking (but not breaking in practice) changes. WDYT of sending this message to clarify what a BREAKING CHANGE in a minor means?

Reply:
This is okay with me IMO